src/index.html

```diff
@@ -12,7 +13,43 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <!-- #endregion -->
+  <body class="page">
+    <!-- <h1 class="hidden">Calendar</h1> -->
```

Comment:
Do you need this commented line?

Reply:
<!-- #region --> and <!-- #endregion --> For convenience only:)